### ISSUE TITLE
`Communication`: Re-add links to Posts

### ISSUE THREAD

**Issue by dfuchss** [ls1intum/Artemis#9058]
### Is your feature request related to a problem?

In the old version of the communication module, it was possible to Link threads. This was really useful to provide students a link to the answer if questions get asked multiple times. Also, in other use cases, we used the feature a lot.

### Describe the solution you'd like

Re-add a feature for linking Threads. E.g., "copy link to Message" in a context menu (similar to slack)

**Comment by krusche** [comment]
I agree, we should implement this feature! Ideally, we also offer to forward messages, e.g. in a similar way as Slack

**Comment by dfuchss** [comment]
Maybe, if we simply show the ID of the message, we can use things similar to https://github.com/ls1intum/Artemis/pull/9566

**Comment by krusche** [comment]
@asliayk is working on an implementation for forward messages

**Comment by PaRangger** [comment]
Forwarding Messages was implemented in #10331.
